jython/DispatcherSystem/Simulation.py

```diff
@@ -371,3 +380,18 @@ def getPrintStatus(self, status):
             return "status = running"
         else:
             return "status = not running"
+
+    def forward_stopping_sensor_exists(self, transit_name):
+        forward_stopping_sensor = self.forward_stopping_sensor(transit_name)
+        if forward_stopping_sensor != None:
+            return True
+        else:
+            return False
+
+    def forward_stopping_sensor(selfself, activeTrain):
+        transit = activeTrain.getTransit()
+        transit_section_list = transit.getTransitSectionList()
+        transit_section = transit_section_list[transit.getMaxSequence()-1]
+        section = transit_section.getSection()
+        forward_stopping_sensor = section.getForwardStoppingSensor()
+        return forward_stopping_sensor
```

Comment:
Static analysis has failed due to that this file doesn't have an empty line at the end of the file.

Reply:
@bobjacobsen 
@bill-fitch03 has fixed this and added an empty line. But GitHub still marks it as the last line is not empty and Static analysis fails. What could be the cause of this? CR+LF instead of LF? There doesn't seem to be any spaces or TABs on the line.

Reply:
I think Bill just fixed it.  There was a tab-space after the new newline, so still no newline as last character in the file.